tutorials/visualisation/eve/geom_lhcb.C

```diff
@@ -21,7 +21,7 @@ void geom_lhcb()
 
    gEve->FullRedraw3D(kTRUE);
 
-   // EClipType not exported to CINT (see TGLUtil.h):
+   // EClipType not exported to CLING (see TGLUtil.h):
```

Comment:
That is probably inaccurate (it was not exported in CINT, it now available through Cling and code could be updated to use the enums).